changelogs/fragments/147_lookup.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
Should this be `bugfixes`? I feel like if it's not behaving the way the ansible documentation says it should behave then it's a bug.

Reply:
Got it. I am checking `errors=ignore` behavior and will update with that.